### PR TITLE
Try official Oracle Linux AWS AMIs

### PR DESCRIPTION
**Pull request by danmyway**
* as per guidance from https://forums.oracle.com/ords/apexds/post/launch-an-oracle-linux-instance-in-aws-9462 it is possible to launch an official Oracle Linux on AWS
* the images were successfully deployed on Testing Farm staging and production, try using in pipeline
* add post install script to enable root login

Jira Issues: [RHELC-](https://issues.redhat.com/browse/RHELC-)

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [ ] `[RHELC-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [ ] PR title explains the change from the user's point of view
- [ ] Code and tests are documented properly
- [ ] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -68,42 +68,152 @@ jobs:
...
+            tags:
+              BusinessUnit: sst_conversions
+  labels:
+    - tier0
+    - oraclelinux-8
+    - non-destructive
+
+- &tests-tier0-destructive-oraclelinux-8
+  <<: *tests-tier0-non-destructive-oraclelinux-8
+  identifier: "tier0-destructive-centos"
```
Comment (kokesak) [inline .packit.yaml:162]:
```suggestion
  identifier: "tier0-destructive-oraclelinux-8"
```

**Review by kokesak — CHANGES_REQUESTED** [review]

File: .packit.yaml [hunk 2]
```diff
@@ -68,42 +68,152 @@ jobs:
...
+    - oraclelinux-7
     - non-destructive
 
-- &tests-tier1-destructive-manual
-  <<: *tests-tier0-destructive-manual
-  identifier: "tier1-destructive"
-  tmt_plan: "tier1/destructive"
+- &tests-tier0-destructive-oraclelinux-7
+  <<: *tests-tier0-non-destructive-oraclelinux-7
+  identifier: "tier0-destructive-centos"
```
[the remark this answers is not in the document]
Comment (danmyway) [reply .packit.yaml]: done, thx

File: .packit.yaml (hunk 1 above)
Comment (danmyway) [reply .packit.yaml]: done, good catch

**Review by kokesak — APPROVED** [review]
LGTM!, if the tests are fine let's merge it!

**Pull request merged**